### DISCUSSION TITLE
Lung Segmentation output very different from label

### DISCUSSION THREAD

**Discussion by jonathanquantumalpha** [Project-MONAI/MONAI#7012] [Q&A]
Hi,

I'm doing lung segmentation from the medical decathlon data for lung. The code works fine without any errors, however the results I'm getting is very different from the labels (see attached image -- basically the output is highlighting the whole lung as tumorous). Here are the current transforms that I have:

```    train_transforms = Compose(
        [
            LoadImaged(keys=["vol", "seg"]),
            AddChanneld(keys=["vol", "seg"]),
            Orientationd(keys=["vol", "seg"], axcodes="RAS"),
            ScaleIntensityRanged(keys=["vol"], a_min=a_min, a_max=a_max, b_min=0.0, b_max=1.0, clip=True),
            CropForegroundd(keys=["vol", "seg"], source_key="vol"),
            Resized(keys=["vol", "seg"], spatial_size=spatial_size),
            ToTensord(keys=["vol", "seg"]),

        ]
    )

    test_transforms = Compose(
        [
            LoadImaged(keys=["vol", "seg"]),
            AddChanneld(keys=["vol", "seg"]),
            Orientationd(keys=["vol", "seg"], axcodes="RAS"),
            ScaleIntensityRanged(keys=["vol"], a_min=a_min, a_max=a_max,b_min=0.0, b_max=1.0, clip=True),
            CropForegroundd(keys=['vol', 'seg'], source_key='vol'),
            Resized(keys=["vol", "seg"], spatial_size=spatial_size),
            ToTensord(keys=["vol", "seg"]),


        ]```

Would appreciate any ideas on how I can improve this. Thanks!
![download (1)](https://github.com/Project-MONAI/MONAI/assets/135941084/d9fb69ce-e35f-48e6-992f-3e00f90d999b)

**Comment by KumoLiu** [comment]
Hi @jonathanquantumalpha, there are many possible reasons.
First I would suggest you follow the [tutorial](https://github.com/Project-MONAI/tutorials/tree/main/3d_segmentation).
After pre-processing, I also suggest you check your input whether makes sense.
Also, you could modify your hyperparameters such as learning rate, optimizer, et.al.
Thanks!
